packages/venia-concept/src/ServiceWorker/Utilities/__tests__/messageHandler.spec.js

```diff
@@ -0,0 +1,323 @@
+import {
+    __handlers__,
```

Comment:
It's usually a ⚠️ to see private variables like this imported in a test.

Sometimes the solution is as easy as "don't test that internal piece" / the tests are too granular, but other times you may have to refactor the code to be more easily unit-testable.

Reply:
In this case, you may want to add a `clearHandlers` (or similar) function on `messageHandler.js` itself and just call that from here? Depends if you think anyone else could benefit from such a function.

Reply:
I modified the code to be able to test. Ideally `__handlers__` is supposed to be an `internal use only` variable but I had to change to make the tests possible. Without exposing `__handlers__` there is no way to test them easily. We will have to go around and mock a lot of stuff. I feel, as a developer, if someone sees `__VARIABLE_NAME__` they need to use caution.